install_config/monitoring/update-and-compatibility-guarantees.adoc

```diff
@@ -0,0 +1,20 @@
+[[update-and-compatibility-guarantees]]
+= Update and compatibility guarantees
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+In order to be able to deliver updates with guaranteed compatibility, configurability of the OpenShift Monitoring stack is limited to the explicitly available options. This document describes known pitfalls of which types of configuration and customization are unsupported, as well as misuse of resources provided by OpenShift Monitoring. All configuration options described in xref:configuring-openshift-cluster-monitoring.adoc#configuring-openshift-cluster-monitoring[Configuring OpenShift cluster monitoring] are explicitly supported. See also xref:configuring-openshift-cluster-monitoring.adoc#supported-configuration[Supported configuration].
+
+== Modifying OpenShift monitoring resources
+
+The OpenShift Monitoring stack ensures its resources are _always_ in the state it expects them to be. If they are modified, OpenShift Monitoring will ensure that this will be reset. Nonetheless it is possible to pause this behavior, by setting the `paused` field in the `AppVersion` called `openshift-monitoring`. Setting the OpenShift Monitoring stack to be paused, stops all future updates and will cause modification of the otherwise managed resources. If resources are modified in an uncontrolled manner, this will cause undefined behavior during updates.
+
+In order to ensure compatible and functioning updates, the `paused` field must be set to `false` on upgrades.
+
+== Using resources created by OpenShift monitoring
+
+OpenShift Monitoring creates a number of resources. These resources are not meant to be used by any other resources, as there are no guarantees about their backward compatibility. For example, a `ClusterRole` called `prometheus-k8s` is created, and has very specific roles that exist solely for the cluster monitoring Prometheus pods to be able to access the resources it requires access to. All of these resources have no compatibility guarantees going forward. While some of these resources may incidentally have the necessary information for RBAC purposes for example, they can be subject to change in any upcoming release, with no backward compatibility.
+
+If the `Role` or `ClusterRole` objects that are similar are needed, we recommend creating a new object that has exactly the permissions required for the case at hand, rather than using the resources created and maintained by OpenShift Monitoring.
```

Comment:
Maybe "Because the OpenShift Monitoring objects do not have guaranteed backwards compatibility, do not reuse these objects."